packages/p2p/src/stores/order-store.js

```diff
@@ -525,6 +548,10 @@ export default class OrderStore {
         this.data = data;
     }
 
+    setErrorCode(error_code) {
```

Comment:
Just a non-blocking suggestion, maybe later during refactoring, we can move all `error_code` and its related observables to general-store, cuz we already have `error_code` observable inside my-ads-store

Reply:
ah yeah, there will be one in general store once one of my cards have been merged, will put that in the list of TODOS